src/guide/code-owners.md

```diff
@@ -170,7 +157,7 @@ This list is intended to make it easier to identify which email list to include
 * `jdk.jdwp.agent` – Serviceability
 * `jdk.jfr` – JFR
 * `jdk.jlink` – LangTools
-* `jdk.jpackage` – Client
+* `jdk.jpackage` – Client / jpackage
```

Comment:
Client Libs?

Reply:
No .- not the client-libs mailing list nor is it part of client libs, even if it is somewhat connected to client
overall as it is primarily for creating client app installers.
and  jpackage is discussed on core-libs as already noted.